### ISSUE TITLE
Last update broke my workflow. UI freezes on interacting with nodes. Clean install dos not help.

### ISSUE THREAD

**Issue by protector131090** [comfyanonymous/ComfyUI#4820]
### Expected Behavior

Normal interacting with nodes. changing prompt dragind the windows

### Actual Behavior

I touch fast group bypasser rhgthree or String Literal - comfy **UI** freezes. Refreshing the page with F5 helps till io touch it again. switching browsers or reinstalling comfy does not help!

### Steps to Reproduce

Touch Prompt (string literal) window, try to drag it. Or try interacting with fast group bypasser. 
[FLux Dev + Upscaler + Adetailer + 2 LORAs. Super clean v3 1 LORA loader (1).json](https://github.com/user-attachments/files/16914854/FLux.Dev.%2B.Upscaler.%2B.Adetailer.%2B.2.LORAs.Super.clean.v3.1.LORA.loader.1.json)
![Снимок экрана (3249)](https://github.com/user-attachments/assets/38ec6955-edc1-4a26-95a9-9f15cf4d8deb)

### Debug Logs

```powershell
S:\ComfyUI\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-09-07 08:41:02.961367
** Platform: Windows
** Python version: 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
** Python executable: S:\ComfyUI\ComfyUI_windows_portable\python_embeded\python.exe
** ComfyUI Path: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI
** Log path: S:\ComfyUI\ComfyUI_windows_portable\comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   1.4 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 24564 MB, total RAM 65291 MB
pytorch version: 2.4.1+cu121
xformers version: 0.0.28.dev895
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
Using xformers cross attention
[Prompt Server] web root: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\web
Adding extra search path checkpoints C:\sd.webui\webui\models/Stable-diffusion
Adding extra search path configs C:\sd.webui\webui\models/Stable-diffusion
Adding extra search path vae C:\sd.webui\webui\models/VAE
Adding extra search path loras C:\sd.webui\webui\models/Lora
Adding extra search path loras C:\sd.webui\webui\models/LyCORIS
Adding extra search path upscale_models C:\sd.webui\webui\models/ESRGAN
Adding extra search path upscale_models C:\sd.webui\webui\models/RealESRGAN
Adding extra search path upscale_models C:\sd.webui\webui\models/SwinIR
Adding extra search path embeddings C:\sd.webui\webui\embeddings
Adding extra search path hypernetworks C:\sd.webui\webui\models/hypernetworks
Adding extra search path controlnet C:\sd.webui\webui\models/ControlNet
S:\ComfyUI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\kornia\feature\lightglue.py:44: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.
  @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
Adding S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes to sys.path
Could not find efficiency nodes
Could not find ControlNetPreprocessors nodes
Could not find AdvancedControlNet nodes
Could not find AnimateDiff nodes
Could not find IPAdapter nodes
Could not find VideoHelperSuite nodes
### Loading: ComfyUI-Impact-Pack (V7.5)
WARNING ⚠️ Ultralytics settings reset to default values. This may be due to a possible problem with your settings or a recent ultralytics package update.
View settings with 'yolo settings' or at 'C:\Users\user\AppData\Roaming\Ultralytics\settings.yaml'
Update settings with 'yolo settings key=value', i.e. 'yolo settings runs_dir=path/to/dir'. For help see https://docs.ultralytics.com/quickstart/#ultralytics-settings.
### Loading: ComfyUI-Impact-Pack (Subpack: V0.6)
### Loading: ComfyUI-Impact-Pack (V7.5)
Loaded ImpactPack nodes from S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack[Impact Pack] Wildcards loading done.

[Impact Pack] Wildcards loading done.
[ComfyUI-Easy-Use] server: v1.2.3 Loaded
[ComfyUI-Easy-Use] web root: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\web_version/v2 Loaded
### Loading: ComfyUI-Impact-Pack (V7.5)
[Impact Pack] Wildcards loading done.
### Loading: ComfyUI-Manager (V2.50.3)
### ComfyUI Revision: 2653 [c27ebeb1] | Released on '2024-09-06'

[rgthree] Loaded 42 exciting nodes.

[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
WAS Node Suite: OpenCV Python FFMPEG support is enabled
WAS Node Suite Warning: `ffmpeg_bin_path` is not set in `S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.
WAS Node Suite: Finished. Loaded 218 nodes successfully.

        "The distance between insanity and genius is measured only by success." - Bruce Feirstein


Import times for custom nodes:
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Custom-Scripts
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
   0.0 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.3 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfy-image-saver
   0.4 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
   1.3 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   1.5 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui
   1.8 seconds: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-art-venture

Starting server

To see the GUI go to: http://127.0.0.1:8188
FETCH DATA from: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
FETCH DATA from: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
FETCH DATA from: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
FETCH DATA from: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
FETCH DATA from: S:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
```

**Comment by christian-byrne** [comment]
It has been fixed upstream:

- Comfy-Org/litegraph.js#126

You can apply that PR manually in the meantime. Change line 67033 in `ComfyUI\web\assets\index-Dfv2aLsq.js` from this:

```javascript
const isCollapsed = shape === LiteGraph.CARD_SHAPE && node.flags.collapsed;
```

to this:

```javascript
const isCollapsed = shape === LiteGraph.CARD_SHAPE;
```

**Comment by protector131090** [comment]
> It has been fixed upstream:
> 
> * [Fix collapsed regression in #114 Comfy-Org/litegraph.js#126](https://github.com/Comfy-Org/litegraph.js/pull/126)
> 
> You can apply that PR manually in the meantime. Change line 67033 in `ComfyUI\web\assets\index-Dfv2aLsq.js` from this:
> 
> ```js
> const isCollapsed = shape === LiteGraph.CARD_SHAPE && node.flags.collapsed;
> ```
> 
> to this:
> 
> ```js
> const isCollapsed = shape === LiteGraph.CARD_SHAPE;
> ```

that did it. Thanks

**Comment by ltdrdata** [comment]
You can apply latest version via this commad line args
`--front-end-version Comfy-Org/ComfyUI_frontend@latest`
